com.unity.render-pipelines.core/Editor/PostProcess/LensFlareEditor.cs

```diff
@@ -0,0 +1,36 @@
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace UnityEngine
+{
+    /// <summary>
+    /// Editor for Flare (builtin): Editor to show an error message
+    /// </summary>
+    [CustomEditorForRenderPipeline(typeof(UnityEngine.Flare), typeof(Rendering.RenderPipelineAsset))]
+    internal class FlareEditor : Editor
+    {
+        /// <summary>
+        /// Implement this function to make a custom inspector
+        /// </summary>
+        public override void OnInspectorGUI()
+        {
+            EditorGUILayout.HelpBox("This asset doesn't work on SRP {URP & HDRP}, use Lens Flare (SRP) instead.", MessageType.Error);
```

Comment:
it is also for a custom render pipeline. I dunno if it worth specifying `{URP & HDRP}`